css/index.css

```diff
@@ -0,0 +1,309 @@
+  border-radius: 17px;
+  padding: 32px;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.img1 {
+  margin-right: 16px;
+}
+
+.block-header {
+  display: flex;
+}
+
+.header-text {
+  display: flex;
+  flex-direction: column;
+}
+.plan-name {
+  font-family: "Poppins";
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 32px;
+  margin-bottom: 9px;
+}
+
+.dollar {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 36px;
+  color: rgba(12, 36, 71, 0.3);
+}
+
+.number {
+  font-family: "Poppins";
+  font-size: 32px;
+  line-height: 48px;
+  font-weight: 700;
+  font-style: normal;
+}
+
+.month {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 36px;
+  color: rgba(12, 36, 71, 0.3);
+}
+.info-container {
+  margin-top: 24px;
+  display: flex;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.note {
+  margin-right: 8px;
+}
+.songs {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  margin-right: 34px;
+}
+
+.songs-2 {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  margin-right: 23px;
+}
+
+.storage {
+  margin-right: 8px;
+  align-items: center;
+}
+
+.memory {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.line {
+  width: 100%;
+  height: 1px;
+  background-color: #e4e8ec;
+}
+
+.list {
+  list-style: none;
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 16px;
+  margin-top: 24px;
+  color: rgba(12, 35, 70, 0.8);
+}
+
+.chek-box {
+  margin-right: 16px;
+}
+
+li {
+  margin-bottom: 16px;
+}
+
+.list-item {
+  display: flex;
+  align-items: center;
+}
+
+.btn-choose {
+  position: absolute;
+  bottom: 32px;
+  background-color: #3983f2;
+  border-radius: 10px;
+  border: none;
+  box-sizing: border-box;
+  padding: 18px 24px 14px 24px;
+  display: flex;
+  align-items: center;
+  box-shadow: 0 5px 0 #3c354a;
+  transition-duration: 1s, 100ms, 100ms;
+  transition-property: background-color, bottom, box-shadow;
+  transition-delay: 0s, 0.5s, 0.5s;
+}
+
+.btn-choose:hover {
+  bottom: 29px;
+  background-color: #41e632;
+  box-shadow: 0px 3px 0px #3c354a;
+  transition-duration: 1s, 100ms, 100ms;
+  transition-property: background-color, bottom, box-shadow;
+  transition-delay: 0s, 0.5s, 0.5s;
+}
+
+.btn-choose-text {
+  color: white;
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  margin-right: 80px;
+}
+
+.arrow {
+  width: 24px;
+  height: 24px;
+}
+
+.Unlimited-song {
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  margin-right: 16px;
+}
+
+.avatar1 {
```

Comment:
looks like avatar1/2/3 have same styles
make one class and use for three elements instead of 3 same classes with same slectors

Reply:
and if you need to change bgimage for different element different; 
make there new class only with bgimages and use two classes

example:
`
.test {
color: red;
}

test1 {
background-image: url(../img/1.svg);
}

test2 {
background-image: url(../img/2.svg);
}

html:
<div class="test test2" />
<div class="test test1" />
`

Reply:
<div class="test test2" />
<div class="test test1" />